.gitignore

```diff
@@ -102,3 +102,4 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+.vscode/settings.json
```

Comment:
I would consider this an issue of ones local dev environment.
Please put files to be ignored in your local repository into `.git/info/exclude`.
I would also recommend to ignore the whole folder.